provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -456,7 +485,10 @@ func generateSchema() schema.PackageSpec {
 							"https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html",
 					},
 					"skipDefaultNodeGroup": {
-						TypeSpec: schema.TypeSpec{Type: "boolean"},
+						TypeSpec: schema.TypeSpec{
+							Type:  "boolean",
+							Plain: true,
+						},
```

Comment:
This is plain in the implementation:

https://github.com/pulumi/pulumi-eks/blob/c6f1fce98fa2683abe7ffbad7f166da99928fb83/nodejs/eks/cluster.ts#L1147-L1151